library/src/main/java/com/bumptech/glide/Glide.java

```diff
@@ -185,13 +185,12 @@ private static void initGlide(Context context) {
           annotationGeneratedModule.getExcludedModuleClasses();
       for (Iterator<GlideModule> iterator = manifestModules.iterator(); iterator.hasNext();) {
         GlideModule current = iterator.next();
-        if (!excludedModuleClasses.contains(current.getClass())) {
-          continue;
+        if (excludedModuleClasses.contains(current.getClass())) {
+          if (Log.isLoggable(TAG, Log.DEBUG)) {
+            Log.d(TAG, "RootGlideModule excludes manifest GlideModule: " + current.getClass());
```

Comment:
Any particular reason to reduce the amount of info logged? Changing to `current.getClass()` loses information in the logs. Consider for example if for some reason there are two modules exists of the same type, seeing the `hashCode` may be most helpful to realize if it's the same object or there are two instances for some reason.